Algorithm/QCAlgorithm.Plotting.cs

```diff
@@ -285,8 +284,7 @@ public void PlotIndicator<T>(string chart, params IndicatorBase<T>[] indicators)
         /// </summary>
         [DocumentationAttribute(Charting)]
         [DocumentationAttribute(Indicators)]
-        public void PlotIndicator<T>(string chart, bool waitForReady, params IndicatorBase<T>[] indicators)
-            where T : IBaseData
+        public void PlotIndicator(string chart, bool waitForReady, params IndicatorBase[] indicators)
```

Comment:
Nice! We could probably merge these too implementation if the above called the bellow with wait for ready false?